niworkflows/interfaces/morphology.py

```diff
@@ -0,0 +1,105 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2021 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+""" Handling brain mask"""
+
+from nipype.interfaces.base import (
+    traits,
+    TraitedSpec,
+    BaseInterfaceInputSpec,
+    File,
+    SimpleInterface,
+)
+
+
+class _CrownMaskInputSpec(BaseInterfaceInputSpec):
+    in_segm = File(
+        exists=True, mandatory=True, position=0, desc="Input discrete segmentation of the brain."
+    )
```

Comment:
```suggestion
```

Reply:
Have you determined that it is useless to take the union of the functional and structural mask ? If not, we cannot remove this argument

Reply:
Well, I think that union must be done, but not by niworkflows because it is not aware of the inputs. MRIQC and/or fMRIPrep may take different routes here to determine what they consider the brain mask that should be used here.

This PR re-purposes the morphology interface to be more general. It doesn't really implement anything about the crown. It rather prepares the framework to implement the crown easily.